src/test/run-make/issue-38237/Makefile

```diff
@@ -1,11 +1,5 @@
 -include ../tools.mk
```

Comment:
This should be changed to 

```makefile
-include ../../run-make-fulldeps/tools.mk
```

(same for other moved `Makefile`s, if they contain this line.)

Reply:
Wait, is `rustdoc` a "fulldeps" dependency?

Reply:
Rustdoc is not currently a fulldeps dependency I believe.

Reply:
@eddyb It was a fulldeps dependency before stage0 supported proc_macros, probably not now. 

Anyway this line is referring to `-fulldeps` only because the configuration `tools.mk` is located in that directory. The `run-make-fulldeps` separation was created quite recently in 7df6f4161cdc13a19216b5f1087081f490f06cdb, previously we only have `run-make`.

Reply:
Ohhh, I see, `-include ../../run-make-fulldeps/tools.mk` is literally a convention because we didn't put `tools.mk` in a more central place. Seems suboptimal but... okay.